### ISSUE TITLE
Release for Plone 5.2.2

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/plone.app.locales#294]
I am working on a Plone 5.2.2 release, see https://github.com/plone/Products.CMFPlone/issues/3119.

Can we have a `plone.app.locales` release this week? Thanks.

**Comment by vincentfretin** [comment]
Hi @mauritsvanrees Sure I'll do it. Next time, please add my handle @vincentfretin so I receive the github notification in my inbox. I read the other github notifications by batch otherwise, so I may not see it right away or may miss it.
I did the procedure change here https://github.com/plone/buildout.coredev/commit/9b83d735e492b32cd08263411dd9c3157fe2ee95

**Comment by mauritsvanrees** [comment]
Great!
Thanks for updating the procedure, that helps.
And thanks for handling so many things with the translations for so many years!

**Comment by vincentfretin** [comment]
Thanks for the kind words. :)

I merged the following PR just now:
- https://github.com/plone/plone.app.discussion/pull/170
- https://github.com/plone/plone.app.theming/pull/185
Can you please create a new release for those two packages? Sorry about that.

I resynced the plone translations just now after #290 was merged and with the changes I did in plone.app.discussion and plone.app.theming. There are 9 new messages to translate mainly for the new Custom CSS feature in plone.app.theming.

**Comment by mauritsvanrees** [comment]
Thanks, I will create those.
Maybe before I do that you can have a look at https://github.com/plone/plone.app.discussion/pull/167? This does not influence `plone.app.locales`, but has to do with translations anyway.

**Comment by mauritsvanrees** [comment]
Never mind the PR from my last comment, it needs more work.

**Comment by vincentfretin** [comment]
As a reminder, you can see the translations stats here:
https://jenkins.plone.org/view/Translations/job/plone-5.2-i18n/

Btw there is still this issue https://github.com/plone/Products.CMFPlone/issues/2897

**Comment by vincentfretin** [comment]
The stats at https://jenkins.plone.org/view/Translations/job/plone-5.2-i18n/ are not properly updated @gforcada @tisto Can you please take a look?
Italian for example is at 100% right now but this is false.

Let's ping the past contributors to update their translation :)
- Catalan (ca) : @allusa
- Italiano (it) : @cekk @ale-rt @pnicolli @arsenico13 
- Basque (eu) : @erral 
- Español (es) : @erral 
- French (fr) : @vincentfretin @laulaz @dbitouze
- Deutsch (de) : @jensens @thet @ksuess
- Slovenian (sl) : @balavec 
- Português do Brasil (pt-br) @LeuAlmeida @hvelarde 
- Russian (ru) : @serge73
- Nederlands (nl) : @fredvd
- Traditional Chinese (zh-tw) : @l34marr
- Chinese Simplified (zh-cn) : @adam139
- Norsk (no) : @espenmn
- Japanese (ja) : @terapyon

**Comment by ale-rt** [comment]
I agree with Maurits about the marvelous work you did in all those years! Thanks! :bowing_man:

**Comment by erral** [comment]
I already updated es and eu.

 I think that the Jenkins job to update the translation stats is set to run once a day.

**Comment by vincentfretin** [comment]
I started a list of known i18n issues here 
https://github.com/plone/Products.CMFPlone/issues/3119#issuecomment-646509534

**Comment by vincentfretin** [comment]
I released plone.app.locales 5.1.22
I can do a new release just before Plone 5.2.2 if there are any translation updates.

**Comment by vincentfretin** [comment]
In prevision to a new release of Plone 4.3.x, if you want some translation updates in Plone 4.3.x, do your changes in plone.app.locales 4.3.x branch now. For now, there is no changes, so no release needed.

**Comment by vincentfretin** [comment]
plone.app.locales 5.1.22 is compatible with Plone 5.1.x as well. I updated the versions.cfg of both buildout.cordev 5.1 and 5.2

**Comment by vincentfretin** [comment]
I reopen so we don't forget to create a new release.

**Comment by vincentfretin** [comment]
Don't forget to update your translation in widgets.po too. Verify all your po files.

**Comment by vincentfretin** [comment]
plone.app.locales 5.1.23 released

**Comment by vincentfretin** [comment]
@erral For the stats, it was an issue in i18ndude https://github.com/collective/i18ndude/pull/81

**Comment by vincentfretin** [comment]
I will need a new release, I fixed the date and hours format for French in the pickadate widget
Please see 
https://github.com/plone/Products.CMFPlone/issues/3123 if your language doesn't have the correct format too!
@espenmn will update the format for Norwegian.

**Comment by vincentfretin** [comment]
For the following issue https://github.com/plone/Products.CMFPlone/issues/3128
I fixed the translation in all languages by replacing `<em>` tag with double quotes except for a few asian languages where I simply removed the `<em>` without adding double quotes, same for some languages where there was a `-something` suffix.
Here is the changes https://github.com/collective/plone.app.locales/commit/9b6b82ca1f1845571f031fd1c8406c365c673a1d
Anyway if you don't like the double quotes in your language, please modify your translation.
You should make sure that what is in double quotes "Edit" correspond to your translation of the Edit tab on the dashboard page, so probably the translation for `label_edit` for example.

**Comment by vincentfretin** [comment]
I added one message in plone.po for the Custom CSS placeholder https://github.com/plone/Products.CMFPlone/issues/3139

**Comment by vincentfretin** [comment]
Please update your translations today.
I'll do a last release for Plone 5.2.2 tomorrow morning CET.

**Comment by vincentfretin** [comment]
The Plone 5.2.2 will be postponed for a least a week, so no hurry for the plone.app.locales release.
I'm good at my job pressuring you to update the translations, right? I know it works, @erral updated them. :-D

**Comment by vincentfretin** [comment]
New messages to translate in widgets.po for the folder contents actions thanks to @erral https://github.com/plone/mockup/pull/992

**Comment by vincentfretin** [comment]
fr, eu and es are at 100%!

Translations almost at 100% :

@balavec
```
sl	: 3937 (98.57%) / 0 / 57 = 3994
	atcontenttypes: 174/0/0 = 174
	atreferencebrowserwidget: 18/0/0 = 18
	cmfeditions: 67/0/0 = 67
	cmfplacefulworkflow: 60/0/0 = 60
	plone: 3317/0/36 = 3353
	plonefrontpage: 11/0/0 = 11
	plonelocales: 49/0/0 = 49
	widgets: 241/0/21 = 262
```

@jensens @ksuess
```
de	: 3935 (98.52%) / 2 / 57 = 3994
	atcontenttypes: 174/0/0 = 174
	atreferencebrowserwidget: 15/0/3 = 18
	cmfeditions: 67/0/0 = 67
	cmfplacefulworkflow: 60/0/0 = 60
	plone: 3305/2/46 = 3353
	plonefrontpage: 11/0/0 = 11
	plonelocales: 49/0/0 = 49
	widgets: 254/0/8 = 262
```

@LeuAlmeida @hvelarde
```
pt-br	: 3862 (96.70%) / 22 / 110 = 3994
	atcontenttypes: 169/0/5 = 174
	atreferencebrowserwidget: 18/0/0 = 18
	cmfeditions: 67/0/0 = 67
	cmfplacefulworkflow: 58/0/2 = 60
	plone: 3282/22/49 = 3353
	plonefrontpage: 11/0/0 = 11
	plonelocales: 49/0/0 = 49
	widgets: 208/0/54 = 262
```

@cekk @ale-rt @pnicolli @arsenico13
```
it	: 3945 (98.77%) / 0 / 49 = 3994
	atcontenttypes: 174/0/0 = 174
	atreferencebrowserwidget: 18/0/0 = 18
	cmfeditions: 67/0/0 = 67
	cmfplacefulworkflow: 58/0/2 = 60
	plone: 3346/0/7 = 3353
	plonefrontpage: 11/0/0 = 11
	plonelocales: 49/0/0 = 49
	widgets: 222/0/40 = 262
```

@allusa
```
ca	: 3904 (97.75%) / 2 / 88 = 3994
	atcontenttypes: 169/0/5 = 174
	atreferencebrowserwidget: 10/0/8 = 18
	cmfeditions: 65/0/2 = 67
	cmfplacefulworkflow: 53/2/5 = 60
	plone: 3320/0/33 = 3353
	plonefrontpage: 11/0/0 = 11
	plonelocales: 49/0/0 = 49
	widgets: 227/0/35 = 262
```

**Comment by hvelarde** [comment]
I'm sorry, I can't help on this.

**Comment by vincentfretin** [comment]
plone.app.locales 5.1.24 released
